tsconfig.base.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "allowJs": true,
+    "checkJs": false,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true,
+
+    // From https://www.typescriptlang.org/docs/handbook/babel-with-typescript.html to use Babel for transpiling, tsc for types
```

Comment:
so sorry if this is silly. but are comments allowed in JSON?

Reply:
ah, nvm, figured it out! ignore my comment ;D https://stackoverflow.com/questions/73743077/why-normal-json-files-dont-allow-comments-where-as-tsconfig-json-allow-comment